mcdreforged/cli/cli_entry.py

```diff
@@ -157,3 +176,121 @@ def write(base_path: str, *, directory_only: bool):
 
 	writeln('Packed {} files/folders into "{}"'.format(file_counter, file_name))
 	writeln('Done')
+
+def init_plugin_workspace(path: str, pid: str, name: str, description: str, authors: str, link: str, resources: list[str], entrypoint: str, archive_name: str, *, quiet: bool = False):
```

Comment:
I don't think stuffs like `list[str]` works in python 3.6. You may want to use `List[str]`

Reply:
emm, I don't know how to use it.

Reply:
use classes in the `typing` package for type annotating instead of raw type with `[]`, or this will only work in python 3.10